liquidity-module/index.md

```diff
@@ -35,22 +35,30 @@ You can follow the [code with us session](https://www.youtube.com/watch?v=Gxaqpz
 
 Before you start the tutorial, install the prerequisite software. 
 
-- [Install Starport](../starport/index.md) v0.16.2
+- [Install Starport](../starport/index.md) v0.16.2 or later
```

Comment:
In order to feature the later `starport chain serve` command we need to make sure to update Starport to v0.17.x here (let's use v0.17.3)